src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -161,14 +161,20 @@ export default App.extend({
 
     channel.request('subscribe', this.collection.models, { filters: { [filterKey]: this.filters } });
 
-    this.listenTo(channel, 'message', (message, model) => {
+    this.listenTo(channel, 'message', ({ category, payload }) => {
+      if (category !== 'ResourceCreated') return;
+
+      const model = Radio.request('entities', 'get:store', payload.resource);
```

Comment:
I wanted to do this ☝️ in the `onMessage()` of the `/js/services/ws.js` [file](https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/services/ws.js#L101-L116).

Specifically I wanted to do something like:

```js
if (data.category === 'ResourceCreated') {
  model = Radio.request('entities', 'get:store', data.payload.resource);
  channel.trigger('message', data, model);
  return;
}
```

But that would break our handling of the `ResourceCreated` ws notification for actions on the patient flow page.

Both of these `ResourceCreated`'s have both a `data.resource` and `data.payload.resource`. So I'm not sure how to differentiate between the two in that `onMessage()` function.